spec/lib/events/web_request_spec.rb

```diff
@@ -71,6 +83,43 @@
       )
     end
 
+    describe 'anonymised_user_agent_and_ip' do
+      context 'when user agent and ip address both present' do
+        it 'is set to a hash of user agent and ip' do
+          expect(web_request.as_json['anonymised_user_agent_and_ip']).to(
+            eq(Digest::SHA2.hexdigest(user_agent + remote_ip)),
```

Comment:
as `user_agent` and `remote_ip` are determined, could we put a literal hash in here rather than mirroring the code inside the class?

Reply:
Sure.

Reply:
[`e9159ef` (#833)](https://github.com/DFE-Digital/find-teacher-training/pull/833/commits/e9159efb9b0275aaba7f87dccc8b4b67c58d4cd0)

Reply:
https://github.com/DFE-Digital/find-teacher-training/pull/833/commits/b43ca16f9483149277e8b098773eac8bc8a4df96